code-journal.html

```diff
@@ -0,0 +1,45 @@
+<html>
+<head>
+    <meta charset="UTF-8" />
+    <title>My Journey into Software Development Blog</title>
+    <link href="styles/style.css" rel="stylesheet">
+</head>
+
+<body>
+    <header> <nav> <ul>
+        <a href = index.html>Home</a><br>
+        <a href = about.html>About me</a><br>
+        <a href = code-journal.html>Code Journal</a><br>
+        <a href = portfolio.html>Portfolio</a><br>       
+    </ul> </nav> </header>
+  
+    <section>
+            <h2 class = highlight>Code Journal</h2>
```

Comment:
classes should be in quotes
```suggestion
            <h2 class="highlight">Code Journal</h2>
```